mlir/lib/Target/LLVMIR/DebugTranslation.cpp

```diff
@@ -281,6 +291,28 @@ llvm::DILocation *DebugTranslation::translateLoc(Location loc,
   return translateLoc(loc, scope, /*inlinedAt=*/nullptr);
 }
 
+// Translate the given DWARF expression metadata to to LLVM.
```

Comment:
```suggestion
```
You can drop the comment on the declaration since there is one on the definition.